unified_targets/docs/Manufacturers.md

```diff
@@ -31,6 +31,7 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |SPRO|Seriously Pro Racing (SP Racing)|http://seriouslypro.com/|
 |TCMM|Shenzhen Xinmumu Science and Technology Co (TCMM)|http://www.tcmm-rc.cn/|
 |TMTR|T-motor|http://uav-en.tmotor.com/|
+|TLON|TALON|https://www.heli-nation.com/|
```

Comment:
Please follow the instructions at https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md